runtime/bindings/python/src/pyopenvino/graph/node.cpp

```diff
@@ -31,7 +31,7 @@ class PyNode : public ov::Node {
 
 namespace py = pybind11;
 
-using PyRTMap = std::map<std::string, std::shared_ptr<ov::Variant>>;
+using PyRTMap = std::map<std::string, ov::Any>;
```

Comment:
There's a definition of such mapping in the `ov::Node` class. Please change this line to `using PyRTMap = ov::Node::RTMap`. It will avoid code duplication and multiple definitions of such mapping.